index.html

```diff
@@ -4,11 +4,297 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css"
+      integrity="sha512-Evv84Mr4kqVGRNSgIGL/F/aIDqQb7xQ2vcrdIwxfjThSH8CSR7PBEakCr51Ck+w+/U6swU2Im1vVX0SVk9ABhg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
     <title>James Larbie</title>
   </head>
   <body>
     <!-- navbar -->
-    <nav></nav>
+    <nav>
+      <p class="wlc">Welcome</p>
+
+      <div class="menu">
+        <div class="close">
+          <i class="fa-solid fa-x"></i>
+        </div>
+        <ul>
+          <li><a href="#hero">Portfolio</a></li>
+          <li><a href="#abt">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+
+      <i class="fa-solid fa-bars"></i>
+    </nav>
+
+    <!-- hero -->
+    <section id="hero">
+      <div class="intro">
+        <h1>Hey there, Im James</h1>
+        <h1>Im a Software Developer</h1>
+      </div>
+
+      <div>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </div>
```

Comment:
- [ ] It would be better semantically if you put this text in a `<p> </p>` element instead of a div.